.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -0,0 +1,60 @@
+name: Babelfish for PostgreSQL Documentation
+description: Propose changes to Babelfish for PostgreSQL Documentation
+title: "[Documentation]: "
+labels: ["documentation", "untriaged"]
+assignees:
+  - gopinathpai
+  - nasbyj
+  - rcv-aws
+  - susanmdouglas-aws
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this documentation improvement.
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: Please enter your mail address for contacting in the case of needing more information.
```

Comment:
How about:

Please provide an email address where we can contact you for more information.

Reply:
We're going to remove all _contact details_ requests in the templates.